### PR TITLE
Fix OIDC signin tool cache

### PR DESCRIPTION
**Pull request by JuliusArmalis**

### CODE REVIEW

**Review by paulius-valiunas — APPROVED** [review]

File: common/config/azure-pipelines/ci.yml [hunk 1]
```diff
@@ -1,68 +0,0 @@
-parameters:
-  - name: shouldPublish
```
Comment (aruniverse) [inline common/config/azure-pipelines/ci.yml:2]: Why have the pipelines been deleted?

Comment (JuliusArmalis) [reply common/config/azure-pipelines/ci.yml:2]: Because it was duplicated inside "common" and ".github". The one used in PRs seems to be in ".github" so I removed unused files to avoid confusion in the future.

Comment (aruniverse) [reply common/config/azure-pipelines/ci.yml:2]: Im pretty sure the one in the github folder is just the ci pipeline that runs on prs on gh;
the ones in the azure-pipelines are used to actually publish the pkg and run the ci on ado.
please revert deleting these 3 files

Comment (JuliusArmalis) [reply common/config/azure-pipelines/ci.yml]: Reverted the deletion

**Review by aruniverse — APPROVED** [review]

File: common/config/rush/command-line.json [hunk 2]
```diff
@@ -40,6 +40,16 @@
       "ignoreMissingScript": false,
       "ignoreDependencyOrder": true,
       "allowWarningsInSuccessfulBuild": true
+    },
+    {
+      "name": "test",
```
Comment (wgoehrig) [inline common/config/rush/command-line.json:45]: Why was this necessary?  I thought `rush test` already worked...

Comment (JuliusArmalis) [reply common/config/rush/command-line.json:45]: No, it was not a command that was setup with rush. In the pipeline we run rush cover instead, but we figured it would still be nice to have rush test available for local testing.

**Review by wgoehrig — APPROVED** [review]
